WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -1552,6 +1586,7 @@ - (void)showCommentsFromSource:(BlogDetailsNavigationSource)source
 {
     [self trackEvent:WPAnalyticsStatOpenedComments fromSource:source];
     CommentsViewController *controller = [CommentsViewController controllerWithBlog:self.blog];
+    controller.navigationItem.largeTitleDisplayMode = UINavigationItemLargeTitleDisplayModeNever;
```

Comment:
I don't think this works, as setting `prefersLargeDisplayTitles` to `false` will make the navigation bar never display large titles even at the root level.

One other way we could approach this, to avoid having to set it on each child, would be to do something like a custom navigation controller:

```
class LargeTitleNavigationController: UINavigationController {
    override func pushViewController(_ viewController: UIViewController, animated: Bool) {
        viewController.navigationItem.largeTitleDisplayMode = .never
        super.pushViewController(viewController, animated: animated)
    }
}
```

What do you think?

Reply:
That sounds good to me. Another similar approach, if we don't want to subclass, would be to use a factory method that assigns `largeTitleDisplayMode = .never` to the newly created instance.

Reply:
I'm having second thoughts about the navigation controller subclass, as I don't think we should rely on knowing where the navigation controller has come from – for example, the split view controller sometimes creates navigation controllers when the view is collapsed and expanded. I'm going to leave the code as-is for now, and we can always re-visit it later.